ui/apps/platform/src/types/group.proto.ts

```diff
@@ -20,4 +22,6 @@ export type GroupProperties = {
     authProviderId: string;
     key: string;
     value: string;
```

Comment:
For the group properties, both `key` and `value` fields can be treated as optional. Only `id` and `authProviderId` are required.

For a bit of background and edge cases, [here's the reference to the validation within the datastore](https://github.com/stackrox/stackrox/blob/master/central/group/datastore/validate.go).